code/modules/surgery/generic_steps.dm

```diff
@@ -86,4 +86,17 @@
 		H.apply_damage(50,"brute","[target_zone]")
 
 	user.visible_message("[user] saws [target]'s [parse_zone(target_zone)] open!", "<span class='notice'>You saw [target]'s [parse_zone(target_zone)] open.</span>")
-	return 1
\ No newline at end of file
+	return 1
+
+//drill bone
+/datum/surgery_step/drill
+	name = "drill bone"
+	implements = list(/obj/item/weapon/surgicaldrill = 100, /obj/item/weapon/pickaxe/drill = 60, /obj/item/mecha_parts/mecha_equipment/drill = 60, /obj/item/weapon/screwdriver = 20)

```

Comment:
If you fucking get a mecha drill and use it for this, you deserve to have the ability.